maat-court-data-api/build.gradle

```diff
@@ -39,7 +39,7 @@ def versions = [
         cloudLocalstackUtilsVersion      : '0.2.23',
         springContractStubRunnerVersion  : '4.0.1',
         sentryBomVersion                 : '7.1.0',
-        crimeCommonsClasses              : "3.4.0",
+        crimeCommonsClasses: '3.21.0',
```

Comment:
Minor comment: could you please reformat build.gradle so it takes care of the spaces ?

Reply:
Fair point, have amended the white space and also replaced speech marks " with quotes ' were permissible (no substitutions required)